client/era/era_service.go

```diff
@@ -267,3 +269,24 @@ func (sc ServiceClient) ListDatabaseInstance(ctx context.Context) (*ListDatabase
 
 	return res, sc.c.Do(ctx, httpReq, res)
 }
+
+func (sc ServiceClient) RegisterDatabase(ctx context.Context, req *RegisterDBInputRequest) (*ProvisionDatabaseResponse, error) {
+	httpReq, err := sc.c.NewRequest(ctx, http.MethodPost, "/databases/register", req)
+	res := new(ProvisionDatabaseResponse)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return res, sc.c.Do(ctx, httpReq, res)
+}
+
+func (sc ServiceClient) UnRegisterDatabase(ctx context.Context, id string, req *UnRegisterDatabaseRequest) (*ProvisionDatabaseResponse, error) {
```

Comment:
Unregister is just delete with different input arguments right ?

Reply:
yes. Removed this method .